README.md

```diff
@@ -8,6 +8,9 @@ To upload results, you will have to register and generate a token on the website
 
 ## Requirements
 gProfiler works on **Linux** and requires **Python 3.6+** to run.
+
+The `nsenter` program needs to be installed for Java profiling. For Debian/Ubuntu, install the `util-linux` package.
```

Comment:
Maybe it worth to mention that `util-linux` in ubuntu 14.04 not include `nsenter`. Or delete the section about Debian/Ubuntu

Reply:
Until we provide a solution for that (either by bringing that `nsenter` with us, or by not using it) - I'll let the users handle the directions as given in https://askubuntu.com/questions/439056/why-there-is-no-nsenter-in-util-linux